app/pages/sections/order-items/catalogue-solutions/order-item/tiered/patient/manifest.json

```diff
@@ -0,0 +1,84 @@
+      }
+    ],
+    "cellInfo": {
+      "recipient": {
+        "dataTestId": "recipient",
+        "classes": "nhsuk-u-padding-top-5"
+      },
+      "quantity": {
+        "question": {
+          "type": "input",
+          "id": "quantity"
+        },
+        "expandableSection": {
+          "dataTestId": "view-section-input-id-practice",
+          "title": "What list size should I enter?",
+          "innerComponent": "Enter the amount you wish to order. This is usually based on each Service Recipient's practice list size to help calculate an estimated price, but the figure can be changed if required.<br><br>As you’re ordering per patient, we've included each practice list size if we have it. If it's not included, you'll need to add it yourself."
+        },
+        "classes": "nhsuk-u-margin-top-5 nhsuk-input--width-10"
+      },
+      "deliveryDate": {
+        "question": {
+          "type": "date",
+          "id": "deliveryDate"
+        },
+        "expandableSection": {
+          "dataTestId": "view-section-input-id-date",
+          "title": "What date should I enter?",
+          "innerComponent": "Enter the planned delivery date you've agreed with the supplier. This is the day that each of the listed Service Recipients will start using this Catalogue Solution.<br><br>We've included the commencement date for this Call-off Agreement, but you can change it if you’ve agreed a different delivery date."
+        }
+      }
+    }
+  },
+  "errorMessages": {
+    "PriceRequired": "Enter a price",
+    "PriceMustBeANumber": "Price must be a number",
+    "PriceMoreThan3dp": "Price must be to a maximum of 3 decimal places",
+    "PriceGreaterThanOrEqualToZero": "Price cannot be negative",
+    "PriceLessThanMax": "Maximum price exceeded, enter a lower price",
+    "PriceGreaterThanListPrice": "You’ve entered a price that is more than the list price, enter a lower price",
+    "QuantityRequired": "Enter a practice list size",
+    "QuantityMustBeANumber": "Practice list size must be a number",
+    "QuantityInvalid": "Practice list size must be a whole number",
+    "QuantityGreaterThanZero": "Practice list size must be greater than zero",
+    "QuantityLessThanMax": "Maximum practice list size exceeded, enter a lower number",
+    "DeliveryDateRequired": "Enter a planned delivery date",
+    "DeliveryDateInvalid": "Enter a planned delivery date in a valid format",
+    "DeliveryDateOutsideDeliveryWindow": "Planned delivery date must be within 42 months from the commencement date for this Call-off Agreement",
+    "DeliveryDateNotReal": "Planned delivery date must be a real date",
+    "DeliveryDateDayRequired": "Planned delivery date must include a day",
+    "DeliveryDateMonthRequired": "Planned delivery date must include a month",
+    "DeliveryDateYearRequired": "Planned delivery date must include a year",
+    "DeliveryDateYearLength": "Year must be four numbers, for example 2020"
+  }
+}
```

Comment:
We don't need to support tiered prices yet so we can remove the code that relates to tiered pricing. However, keep a copy (either in a stash and/or a copy of the files) as we may be able to make use of it later.

Reply:
removed tiered folder  also kept a copy of the files.